Formula/xtrv.rb

```diff
@@ -0,0 +1,16 @@
+class Xtrv < Formula
+  desc "A simple archiving utility"
+  homepage ""
+  url "https://github.com/ttshivhula/xtrv/archive/v0.0.3.tar.gz"
+  sha256 "f92274fbd719ffd1e43cfe1a1b37fd81d0692fb67cb37b7f131d64cca738a9a1"
+  head "https://github.com/ttshivhula/xtrv.git"
+
+  def install
+    system "make"
+    system "mkdir #{prefix}/bin"
+    system "mkdir #{prefix}/share"
+    system "cp xtrv #{prefix}/bin/"
+    system "cp unxtrv #{prefix}/bin/"
+    system "cp -rf man #{prefix}/share"
+  end
+end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.